### DISCUSSION TITLE
Return to last viewed screen?

### DISCUSSION THREAD

**Discussion by hate29** [xournalpp/xournalpp#3361] [Q&A]
First of all, thanks to everyone of this great software!

Now, is there any way to return to last viewed screen? I mean, when you go to look tables, sources etc. from the last pages of PDF and try to return to where you were? I know that in Okular there is this kind of solution but I think Xournal needs this too, if it’s not possible yet.

**Comment by Technius** [comment]
Not at the moment, but this would be a nice feature. Please submit a feature request.
